transformers4rec/torch/trainer.py

```diff
@@ -309,6 +308,31 @@ def get_scheduler(
             optimizer, num_warmup_steps=num_warmup_steps, num_training_steps=num_training_steps
         )
 
+    def compute_loss(self, model, inputs, return_outputs=False):
+        """
+        Overriding :obj:`Trainer.compute_loss()`
+        To allow for passing the targets to the model's forward method
+        How the loss is computed by Trainer. By default, all Transformers4Rec models return
+        a dictionary of three elements {'loss', 'predictions', and 'labels}
+        """
+        inputs, targets = inputs
+        outputs = model(inputs, targets=targets, training=True)
+        # Save past state if it exists
```

Comment:
This comment was copied from the HF original method `compute_loss` to track the #TODO if we upgrade the HF version.